CommunityToolkit.Mvvm/Input/AsyncRelayCommand.cs

```diff
@@ -323,4 +330,13 @@ public void Cancel()
             PropertyChanged?.Invoke(this, IsCancellationRequestedChangedEventArgs);
         }
     }
+
+    /// <summary>
+    /// Awaits an input <see cref="Task"/> and throws an exception on the calling context, if the task fails.
+    /// </summary>
+    /// <param name="executionTask">The input <see cref="Task"/> instance to await.</param>
+    internal static async void AwaitAndThrowIfFailed(Task executionTask)
```

Comment:
Can we add some notes regarding the need for `async void` here?

Reply:
Done in https://github.com/CommunityToolkit/dotnet/pull/292/commits/612055be6f69a057d6774f60905183828634a669.